ui/components/header-footer.js

```diff
@@ -67,30 +69,30 @@ function faviconTags() {
 export default function HeaderFooter({ children, showSearch, wrapperClassName }) {
   const klasses = ['container', wrapperClassName].join(' ');
 
-  return [
-    <Head key="head">
-      <title>OMB Policy Library (Beta)</title>
-      <link rel="stylesheet" href="/static/styles.css" />
-      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.css" integrity="sha384-8QOKbPtTFvh/lMY0qPVbXj9hDh+v8US0pD//FcoYFst2lCIf0BmT58+Heqj0IGyx" crossOrigin="anonymous" />
-      <script
-        async
-        type="text/javascript"
-        id="_fed_an_ua_tag"
-        src="https://dap.digitalgov.gov/Universal­Federated­Analytics­Min.js?agency=EOP&subagency=OMB"
-      />
-      { faviconTags() }
-      <meta name="viewport" content="width=device-width, initial-scale=1" />
-    </Head>,
-    <div key="page-header-wrapper" className="page-header-wrapper">
-      <Disclaimer key="disclaimer" />
-      <Navbar key="navbar" showSearch={showSearch} />
-    </div>,
-    <div key="body" className={klasses}>
-      {children}
-    </div>,
-    <Footer key="footer" />,
-    <script key="footer-script" src="/static/ie.js" />,
-  ];
+  return (
+    <React.Fragment>
+      <Head>
+        <link rel="stylesheet" href="/static/styles.css" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.css" integrity="sha384-8QOKbPtTFvh/lMY0qPVbXj9hDh+v8US0pD//FcoYFst2lCIf0BmT58+Heqj0IGyx" crossOrigin="anonymous" />
+        <script
+          async
+          type="text/javascript"
+          id="_fed_an_ua_tag"
+          src="https://dap.digitalgov.gov/Universal­Federated­Analytics­Min.js?agency=EOP&subagency=OMB"
+        />
+        { faviconTags() }
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      { pageTitle() }
```

Comment:
Interesting, so `<Head><title key="title">` is defined here, but it's also potentially "overridden" by the `{children}` embedded at line 91 as per next's documentation on [populating `<head>`](https://github.com/zeit/next.js/#populating-head), is that right?

I wonder if it's worth mentioning this in a comment above this line? Up to you really; I'm sure it's something that next.js veterans find idiomatic, while folks unfamiliar with it might mistake as a bug at first.

Reply:
You got it! I'll add a comment to make it clearer.